.claude/commands/review.md

```
@@ -45,6 +45,34 @@ Providers:
+**Before ANY provider dispatch** (whether via orchestrate.sh or manual fallback), resolve the correct model:
+
+```bash
+# Resolve the configured model for a provider:
+PLUGIN_DIR="$(find ~/.claude/plugins -name 'octo-dispatch.sh' -path '*/scripts/*' 2>/dev/null | head -1 | xargs dirname)"
+CODEX_MODEL=$("$PLUGIN_DIR/octo-dispatch.sh" --resolve codex)
+GEMINI_MODEL=$("$PLUGIN_DIR/octo-dispatch.sh" --resolve gemini)
+QWEN_MODEL=$("$PLUGIN_DIR/octo-dispatch.sh" --resolve qwen)
+```
+
+**If you must dispatch manually** (e.g., the review target is not a code diff), use `octo-dispatch.sh`:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)